src/declarai/_base.py

```diff
@@ -78,6 +78,7 @@ class BaseChat(BaseTask):
     """
 
     operator: BaseChatOperator
+    "The operator to use for the chat task."
```

Comment:
Assuming this should be a comment instead of a string? 🙂

Reply:
its a way to document attributes - https://declarai.com/reference/declarai/_base/#declarai._base.BaseTask